src/state/transactions/updater.tsx

```diff
@@ -112,11 +112,6 @@ export default function Updater(): null {
                 hash,
                 isL2 ? L2_TXN_DISMISS_MS : DEFAULT_TXN_DISMISS_MS
               )
-
-              // the receipt was fetched before the block, fast forward to that block to trigger balance updates
-              if (receipt.blockNumber > lastBlockNumber) {
```

Comment:
Can we keep this logic by just adding a jotai hook to fast forward block number

Reply:
Done.